src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx

```diff
@@ -481,17 +489,17 @@ const AwardAmountsChart = ({ awardType, awardOverview, spendingScenario }) => {
                     value: awardAmounts.subsidyAbbreviated,
                     text: 'Original Subsidy Cost',
                     tooltipData: getTooltipPropsByAwardTypeAndSpendingCategory('loan', 'subsidy', awardAmounts),
-                    color: '#F5A623'
+                    color: subsidyColor
```

Comment:
I'm noticing a change here in `subsidyColor` from `#F5A623` to `#0B424D`. Is that intentional?

Reply:
Yes. This was actually part of the ticket. I also had to confirm this with @marcyheld. The color change was deemed appropriate because of the contrast introduced with this color when fileC spending is present. Green and purple is better than yellow and purple.